### PR TITLE
improve flash att perf

### PR DESCRIPTION
**Pull request by pathfinder-pf**
## Motivation
Try to reduce VPU usage and increase block_q size

## Modifications
remove tile operator and change block_q size to 256

## Accuracy Tests
```
JAX_COMPILATION_CACHE_DIR=/tmp/jit_cache python3 -u -m sgl_jax.launch_server --multimodal --model-path=/models/Wan-AI/Wan2.1-T2V-1.3B-Diffusers --precompile-token-paddings=8192 --precompile-bs-paddings=64 --vae-decode-precompile-width-height 480*832 --vae-decode-precompile-frame-paddings 11 41
```
```
time curl http://localhost:30000/api/v1/videos/generation -H "Content-Type: application/json" -d '{"prompt": "A curious raccoon", "size": "480*832", "num_frames": 41, "num_inference_steps": 50}'
```

https://github.com/user-attachments/assets/dea1768d-17df-428c-8f92-398da51fbec1

## Benchmarking and Profiling
1st 1m21.401s
2nd 58.241s

## Checklist

- [x] Please use English, otherwise it will be closed.
- [x] The purpose of the PR, or link existing issues this PR will resolve.
- [x] The test plan, such as providing test command.
- [x] (Optional) The necessary documentation update.

### CODE REVIEW

File: python/sgl_jax/srt/multimodal/kernels/flash_attention.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def get_default(cls, batch_size, num_heads, q_seq_len, kv_len, d_model):
         # TODO(apaszke,sharadmv): Select better parameters based on a heuristic.
         del batch_size, num_heads, q_seq_len, kv_len, d_model  # Unused.
         return BlockSizes(
-            block_q=128,
+            block_q=256,
```
Comment (pengchengneo) [inline python/sgl_jax/srt/multimodal/kernels/flash_attention.py:99]: why change 128 to 256, 256 is the best choice ?

Comment (pathfinder-pf) [reply python/sgl_jax/srt/multimodal/kernels/flash_attention.py:99]: yes, tflops usage can improve 2%
